tests/integration/data_transformer/contracts.py

```diff
@@ -0,0 +1,72 @@
+DATA_TRANSFORMER_UINT256_CONTRACT = """%lang starknet
```

Comment:
```suggestion
DATA_TRANSFORMER_UINT256_CONTRACT = """
%lang starknet
```

does this work? if not, can you put `.strip()` there? or `"""\`